### ISSUE TITLE
Blank line removal incompatible with black

### ISSUE THREAD

**Issue by AntoineD** [PyCQA/docformatter#176]
If a class docstring contains a line that is too long and cannot be reformatted, passing flake8 can be done by appending a `#noqa` to the docstring.
The following was not modified with` docformatter` 1.5.1 and `black` when triggered by `pre-commit`:
```python
class C:
    """Class.""" #noqa

    attr: int
    """Attr."""
```

With `docformatter` 1.6.1, this is changed to:
```python
class C:
    """Class.""" #noqa
    attr: int
    """Attr."""
```

and `pre-commit` always fails because `docformatter` and `black` both apply opposite changes.

This behavior is triggered by the comment appended to the docstring.

**Comment by AntoineD** [comment]
Thank you very much!
When do you plan to release it?

**Comment by weibullguy** [comment]
Well, if I hadn't forgotten to `git pull` on the master branch after merging the PR that fixed this problem, it would've been part of v1.6.1! 🤦🏻‍♂️   I just released v1.6.2 with this fix.
